tests/components/smartthings/conftest.py

```diff
@@ -112,6 +112,7 @@ def mock_smartthings() -> Generator[AsyncMock]:
         "centralite",
         "da_ref_normal_000001",
         "da_ref_normal_01011",
+        "da_ref_normal_01011_onedoor",
```

Comment:
Have you considered loading all the devices in the same mock to reduce parametrization?

Reply:
I need to look into that as you yielded nice results